fahrschein-e2e-test/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>okhttp</artifactId>
             <version>4.9.3</version>
         </dependency>
+        <dependency>
```

Comment:
As far as I can see this is only required for tests, can you then please scope it for tests accordingly?

Reply:
yep, this project only has test anyway, I'll make everything a test dependency.